architecture/core_objects/kubernetes_model.adoc

```diff
@@ -216,26 +216,23 @@ For more on services, see the https://github.com/GoogleCloudPlatform/kubernetes/
 == Namespace
 
 A single cluster should be able to satisfy the needs of multiple user communities.
-
 Each user community wants to be able to work in isolation from other communities.
-
 Each user community has its own:
 
-1. resources (pods, services, replication controllers, etc.)
-2. policies (who can or cannot perform actions in their community)
-3. constraints (this community is allowed this much quota, etc.)
+* Resources: pods, services, replication controllers
+* Policies: who can or cannot perform actions in their community
+* Constraints: this community is allowed this much quota
```

Comment:
Just a suggestion: For these sorts of lists of terms/definitions, I've been using labeled lists with the `[horizontal]` style (http://asciidoctor.org/docs/asciidoc-writers-guide/#labeled-lists and scroll a little).